### ISSUE TITLE
visualization not working - reopen issue #66

### ISSUE THREAD

**Issue by Martin-int01** [iobroker-community-adapters/ioBroker.logparser#97]
Hi,
reopen issue #66
reoccurred in version 2.2.0
the log messages appears in the filter json. The visualization json remains empty.
Already tried to reset the adapter.
thank you in advance

**Comment by ciddi89** [comment]
Yes I know about this issue. Will take an eye of it as soon as possible. Thanks for your patience.

**Comment by Martin-int01** [comment]
ok. thank you for the feedback!
best regards
Martin

**Comment by Martin-int01** [comment]
Hi @ciddi89,

do you have in the meantime a workaround for this issue?

thank you in advance

best regards

Martin

**Comment by ciddi89** [comment]
@Martin-int01 Hi Martin,

Will have an eye of it the next days. Maybe I'll found out what the problem.

**Comment by Martin-int01** [comment]
@ciddi89 
Hi Christian,

great!! Thank you very much in advance!!

**Comment by ciddi89** [comment]
I'm still testing. But it's working correctly at the moment. But I'll keep at it and hopefully get back to you soon if I've found out anything.

**Comment by Martin-int01** [comment]
Hi Christian,

sorry, I've forgotten to give you an important info. 
table0 is always working. 
I've 7 tables. table0-6.  Table0 is working without problem. all other does not work.

Here my configuration:
![image](https://github.com/iobroker-community-adapters/ioBroker.logparser/assets/59722496/06971b78-5755-449f-84b4-198c76e181a4)


thank you in advance

best regards

Martin

**Comment by ciddi89** [comment]
Denke habe den Fehler gefunden und konnte ihn beseitigen. Bitte latest v2.2.1 testen. 

I think I found the error and was able to fix it. Please test latest v2.2.1.

**Comment by Martin-int01** [comment]
Hallo Christian,

sieht sehr gut aus. Ich werde weiter testen.
Vielen Dank und Frohe Weihnachten!!

lg

Martin
